package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "reaction-next-starterkit",
-  "version": "2.0.0-rc.9",
+  "name": "example-storefront",
+  "version": "2.0.0",
```

Comment:
@spencern not sure if I should change this or not

Reply:
Changing the name is fine but I don't know that we should change the version yet. That's part of the release checklist.